api/elections.go

```diff
@@ -661,8 +662,12 @@ func (a *API) electionFilterPaginatedHandler(msg *apirest.APIdata, ctx *httprout
 		}
 	}
 	page = page * MaxPageSize
+
 	// get election list
-	elections, err := a.indexer.ProcessList(body.OrganizationID, page, MaxPageSize, body.ElectionID.String(), 0, 0, body.Status, body.WithResults)
+	if body.WithResults == nil {
+		body.WithResults = new(bool)
```

Comment:
If you need a default value of `false` for WithResults, you might simply declare it as a non pointer variable, you will get default false for free :)